ale_linters/python/mypy.vim

```diff
@@ -6,7 +6,7 @@ let g:ale_python_mypy_args = get(g:, 'ale_python_mypy_options', '')
 function! g:ale_linters#python#mypy#GetCommand(buffer) abort
     return g:ale#util#stdin_wrapper
     \   . ' .py mypy --show-column-numbers '
-    \   . g:ale_python_mypy_options
+    \   . g:ale_python_mypy_args
```

Comment:
let g:ale_python_mypy_args = get(g:, 'ale_python_mypy_options', '')

complies with #257, right? Or do you want to change ale_python_mypy_args to ale_python_mypy_options as well?